custom/panel_templates/Default/assets/css/custom.css

```diff
@@ -1,65 +1,77 @@
 /* --------------- Prevents light flicker on dark mode --------------- */
 
 body {
-    display: none;
+    display: none
```

Comment:
Can we keep semi-colons at the end of lines please?

Reply:
We can... my only issue with that is shipping it without the semi-colons saves users a small amount of code they need to download, and for users with slow connections the smallest amount of code can make a decent difference. Should I add them back or no?

Reply:
the real solution to that problem would be minimizing these files for the upgrade and install packages

Reply:
That would be a great addition :D 
I would still like to merge this in some form though because improvements like the variables